streaming/src/java/org/apache/hive/streaming/HiveStreamingConnection.java

```diff
@@ -520,6 +523,28 @@ private void validateTable() throws InvalidTable, ConnectionError {
       LOG.error(errMsg);
       throw new ConnectionError(errMsg);
     }
+
+    // batch size is only used for managed transactions, not for unmanaged single transactions
+    if (transactionBatchSize > 1) {
+      try (FileSystem fs = tableObject.getDataLocation().getFileSystem(conf)) {
+        if (BlobStorageUtils.isBlobStorageFileSystem(conf, fs)) {
+          // currently not all filesystems implement StreamCapabilities, while FSDataOutputStream does
+          Path path = new Path(tableObject.getDataLocation(), "tmp_stream_verify_" + UUID.randomUUID().toString());
```

Comment:
I am not sure if putting files under table location is a good idea. Better place would be use the /tmp location but if that is not accessible for some reason atleast the files created under the table location should start with "." or "_" so that file listing ignores such files.

Reply:
thank you , I was a bit hesitant on using /tmp, but found enough usage of it in the codebase to use that with enough confidence.